db/migrate/20180621221240_create_votes.rb

```diff
@@ -1,13 +1,14 @@
 class CreateVotes < ActiveRecord::Migration[5.2]
   def change
     create_table :votes do |t|
-      t.references :article, foreign_key: true
+      t.integer :voteable_id, null: false
+      t.string :voteable_type, null: false
```

Comment:
Generally, never modify an existing migration. Never. Never. Never. Even in the cases where it's ok to modify existing migrations, it's never a going to hurt to create a new migration.

If you're looking back at migrations for reference, look at the `schema.rb` instead.